### PR TITLE
Changing quotes of AlCa matrix to be JSON valid

### PR DESCRIPTION
**Pull request by jonuuukas**
changed the matrix dict attributes from single qoutes to double quotes to be JSON valid

### CODE REVIEW

**Comment by mmusich** [comment]
please test

File: Configuration/AlCa/python/autoAlca.py [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
+                  "JetHT"          : "HcalCalDijets+HcalCalIsoTrkFilter+HcalCalIsolatedBunchFilter",
+                  "MET"            : "HcalCalNoise",
+                  "MinimumBias"    : "SiStripCalMinBias+TkAlMinBias",
+                  "MuOnia"         : "TkAlJpsiMuMu+TkAlUpsilonMuMu",
+                  "MuOniaParked"   : "TkAlJpsiMuMu+TkAlUpsilonMuMu",
+                  "SingleElectron" : "EcalCalWElectron+EcalUncalWElectron+EcalCalZElectron+EcalUncalZElectron+EcalESAlign+HcalCalIterativePhiSym+HcalCalIsoTrkFilter",
+                  "SingleMu"       : "MuAlCalIsolatedMu+MuAlOverlaps+TkAlMuonIsolated+DtCalib+MuAlZMuMu+HcalCalHO",
+                  "SingleMuon"     : "TkAlMuonIsolated+DtCalib+MuAlCalIsolatedMu+MuAlOverlaps+MuAlZMuMu+HcalCalIterativePhiSym",
+                  "SinglePhoton"   : "HcalCalGammaJet",
+                  "ZeroBias"       : "SiStripCalZeroBias+TkAlMinBias+LumiPixelsMinBias+SiStripCalMinBias",
```
Comment (ghellwig) [inline Configuration/AlCa/python/autoAlca.py]: @jonuuukas from the title of the PR I would expect that you change only from single to double quotes, but here you are removing '+EcalTrg'. Is that intended?

Comment (mmusich) [inline Configuration/AlCa/python/autoAlca.py]: @ghellwig sure it is not.

**Comment by mmusich** [comment]
-1

File: Configuration/AlCa/python/autoAlca.py (hunk 1 above)
Comment (jonuuukas) [inline Configuration/AlCa/python/autoAlca.py]: that is not intended, sorry. fixing ASAP

**Comment by mmusich** [comment]
please test

**Comment by ghellwig** [comment]
+1

**Comment by mmusich** [comment]
@jonuuukas can you elaborate what is the rationale behind the proposed changes? Is it for usage within CMSSW?

**Comment by jonuuukas** [comment]
JSON valid configurations are needed for the data management app I'm currently developing. As single quotes are not supported, @anorkus suggested to change this file

**Pull request merged**